core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -32,6 +32,13 @@ import { ThreadMessage } from 'domain/feedback_message/ThreadMessage.model';
 import { AppConstants } from 'app.constants';
 import constants from 'assets/constants';
 import { ListSchema, UnicodeSchema } from 'services/schema-default-value.service';
+import { UserContributionRightsDataBackendDict } from 'services/user-backend-api.service';
+
+type Contribution = [string, ActiveContributionDict];
```

Comment:
This seems weird -- the two types are incongruous. What is a "Contribution", exactly?

Reply:
simply a array with two elements, element 0 & 1 respectively have their different type.